### DISCUSSION TITLE
Parcel incorrectly removes type="module" from script tags, breaking the webapp

### DISCUSSION THREAD

**Discussion by gcleaves** [parcel-bundler/parcel#9720] [Q&A]
Hi. Parcel's development build/serve process is removing `type="module"` from script tags. The production build process does not remove the property and works correctly. How can I stop the dev server from making this mistake?

I've tried different --target options when running `npx parcel folder/page.html` but none work. Tips appreciated!

**Comment by Diegovsky** [comment]
hey, I've been stumbling upon this. Have you found any fix? if not, please reference this an issue

**Reply by devongovett** [reply]
`type="module"` is not removed as of `v2.14.0`

**Reply by Kapelianovych** [reply]
> `type="module"` is not removed as of `v2.14.0`

Hi. This does not seem to be true. [This](https://repl.parceljs.org/#JTdCJTIyZmlsZXMlMjIlM0ElNUIlNUIlMjIlMkZzcmMlMkZpbmRleC5odG1sJTIyJTJDJTdCJTIydmFsdWUlMjIlM0ElMjIlM0NzY3JpcHQlMjB0eXBlJTNEJ21vZHVsZSclM0UlNUNuJTIwJTIwY29uc29sZS5sb2coMSklNUNuJTNDJTJGc2NyaXB0JTNFJTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlN0QlNUQlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyZW50cmllcyUyMiUzQSU1QiU1RCUyQyUyMm1pbmlmeSUyMiUzQWZhbHNlJTJDJTIyc2NvcGVIb2lzdCUyMiUzQWZhbHNlJTJDJTIyc291cmNlTWFwcyUyMiUzQWZhbHNlJTJDJTIycHVibGljVXJsJTIyJTNBJTIyJTJGX19yZXBsX2Rpc3QlMjIlMkMlMjJ0YXJnZXRUeXBlJTIyJTNBJTIyYnJvd3NlcnMlMjIlMkMlMjJ0YXJnZXRFbnYlMjIlM0FudWxsJTJDJTIyb3V0cHV0Rm9ybWF0JTIyJTNBJTIyZXNtb2R1bGUlMjIlMkMlMjJobXIlMjIlM0FmYWxzZSUyQyUyMm1vZGUlMjIlM0ElMjJkZXZlbG9wbWVudCUyMiUyQyUyMnJlbmRlckdyYXBocyUyMiUzQWZhbHNlJTJDJTIydmlld1NvdXJjZW1hcHMlMjIlM0FmYWxzZSUyQyUyMmRlcGVuZGVuY2llcyUyMiUzQSU1QiU1RCUyQyUyMm51bVdvcmtlcnMlMjIlM0EwJTdEJTJDJTIydXNlVGFicyUyMiUzQWZhbHNlJTJDJTIyYnJvd3NlckNvbGxhcHNlZCUyMiUzQSU1QiU1RCUyQyUyMnZpZXdzJTIyJTNBJTVCJTIyJTJGc3JjJTJGaW5kZXguaHRtbCUyMiUyQyUyMk9wdGlvbnMlMjIlNUQlMkMlMjJjdXJyZW50VmlldyUyMiUzQTElN0Q=) is reproduction of the issue using Parcel Repl. At the moment of writing Repl uses Parcel v2.14.4 ([commit](https://github.com/parcel-bundler/parcel/commits/c09fee751b61e2f9d52164f88248b9c668614134)) and after building the `index.html` file the `type` attribute is gone. I also tested it on Parcel 2.15.2 and the issue still persists.
